airflow-core/src/airflow/ui/src/components/Clear/Run/ClearRunDialog.tsx

```diff
@@ -101,7 +102,23 @@ const ClearRunDialog = ({ dagRun, onClose, open }: Props) => {
             />
           </Flex>
           <ActionAccordion affectedTasks={affectedTasks} note={note} setNote={setNote} />
-          <Flex justifyContent="end" mt={3}>
+          <Flex
+            {...(dagRun.bundle_version !== null && dagRun.bundle_version !== ""
```

Comment:
We should hide it (or precheck and disable with a tooltip) if it's already the latest version?

Reply:
Have tried to work it out but it's difficult and would involve a lot of code to check if the bundle version is the latest for the bundle